pkgs/by-name/tl/tlock/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "tlock";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "eklairs";
+    repo = "tlock";
+    rev = "v${version}";
+    hash = "sha256-O6erxzanSO5BjMnSSmn89w9SA+xyHhp0SSDkCk5hp8Y=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = "sha256-G402CigSvloF/SI9Wbcts/So1impMUH5kroxDD/KKew=";
+
+  excludedPackages = [
+    "bubbletea"
+    "scripts"
+  ];
```

Comment:
```suggestion
```
Why aren't these being packaged?

Reply:
1. `scripts` - This folder contains bash scripts for development stuff so there is no need for it to be packaged.